server/test/handlers/inputSchedule.test.js

```diff
@@ -1,22 +1,39 @@
-const timeslotHandler = require('../../models/handlers/timeslot');
+const timeslotHandler = require('../../models/handlers/workerTimeslot');
 
 describe('testing inputting of worker schedule functionality', () => {
     beforeEach(() => {
         jest.resetModules(); // Clears any cache between tests.
     });
 
-    test('testing initial stub for inputting worker schedule', async () => {
+    test('testing valid values for inputting worker schedule ', async () => {
         // Arrange
-        const testWorkerId = 8000000;
-        const testSchoolId = 1;
-        const testSlotId = 1;
+        const testWorkerId = '8000000';
+        const testSchoolId = '1';
+        const testSlotId = '1';
+        const testDate = '2020-12-28';  
+        const testStatus = 'available';    
 
         // Act
-        const isSuccessfullyInserted = await timeslotHandler.addWorkerTimeslot(testWorkerId, testSchoolId, testSlotId);
+        const isSuccessfullyInserted = await timeslotHandler.addWorkerTimeslot(testSlotId, testSchoolId, testWorkerId, testStatus, testDate);
 
         // Assert
         expect(isSuccessfullyInserted).toBe(true);
     });
+
+    test('testing invalid values for inputting worker schedule ', async () => {
+        // Arrange
+        const testWorkerId = null ;
+        const testSchoolId = '1';
+        const testSlotId = '1';
+        const testDate = '2020-12-28'; 
+        const testStatus = 'available';       
+
+        // Act
+        const isSuccessfullyInserted = await timeslotHandler.addWorkerTimeslot(testSlotId, testSchoolId, testWorkerId, testStatus, testDate);
+
+        // Assert
+        expect(isSuccessfullyInserted).toBe(false);
+    });
 });
 
 //This is a test code for when the logic of the backend has been successfully implemented for this functionality 
```

Comment:
Hi Peter,
I would recommend reviewing the commented lines below and remove if unneeded (to avoid keeping unnecessary code and increase code clarity).

Reply:
Completed!